backend/src/middlewares/validation/validation.middleware.ts

```diff
@@ -0,0 +1,25 @@
+import { RequestHandler } from 'express';
+import { ValidationSchema } from '~/common/types/types';
+import { HttpError } from '~/exceptions/exceptions';
+import { HttpCode } from '~/common/enums/enums';
+
+const validateSchema = (schema: ValidationSchema): RequestHandler => {
+  const handler: RequestHandler = (req, _res, next) => {
+    const { error } = schema.validate(req.body, {
+      abortEarly: false,
+    });
```

Comment:
```suggestion
    const { error } = schema.validate(req.body);
```
for now we don't want it